documentation/homework/homeworkweek1/App.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const ListItem = props => {
+  const { description, deadline } = props.listInfo;
+  return (
+    <li>
+      {' '}
+      {description},{deadline}
+    </li>
+  );
+};
+
+const StaticList = props => {
+  return (
+    <div className="static-list">
+      <p>Static List</p>
+
+      <ListItem listInfo={{ description: ' Get out of bed', deadline: 'Wed Sep 13 2017' }} />
+      <ListItem listInfo={{ description: 'Brush teeth', deadline: 'Thu Sep 14 2017' }} />
+      <ListItem listInfo={{ description: ' Eat breakfast', deadline: 'Fri Sep 15 2017' }} />
```

Comment:
you could use a `ul` instead of the `div` and move the paragraph out side the `ul` making a valid html tree structure. :)

this comment applies here and on the `DynamicList` Component.